docs/17-porting-tips.md

```diff
@@ -9,7 +9,7 @@ you might have added to your `rc.lua`.
 This avoids most of the possible errors due to missing important changes.
 
 If you still wish to ignore this advice, first read the
-<a href="89-NEWS.md.html#v4">NEWS</a> section about the breaking changes. This
+[NEWS](89-NEWS.md) section about the breaking changes. This
```

Comment:
As far as I know, GitHub doesn't cancel out reverted commits, you have to remove the commits. You can do that using `git rebase -i origin/master; git push --force sudo-nice master`. In the rebase file, just remove the lines. As I told someone yesterday, I have a little script to makes this easier, but in your case, it's pretty straight forward in the first place. 
https://github.com/awesomeWM/awesome/pull/1510#issuecomment-276990321